### ISSUE TITLE
latest dev, no release

### ISSUE THREAD

**Issue by Muzurio** [PokemonGoF/PokemonGo-Bot#1230]
### Expected Behavior
I expect to have pokemons released at startup (value is 850) old bot would release all lower then 850. But also during catching i expect them to be released under 500 or 0.8. Nut nothing is released at all

My config (leaving out the credentials)
```
   "max_steps": 5,
    "mode": "all",
    "walk": 4.01,
    "debug": false,
    "test": false,
    "initial_transfer": 850,
    "location_cache": false,
    "distance_unit": "km",
    "reconnecting_timeout": 15,
    "item_filter": {
      "1": { "keep" : 100 },
      "101": { "keep" : 10 },
      "102": { "keep" : 10 },
      "103": { "keep" : 10 },
	  "103": { "keep" : 10 },
      "201": { "keep" : 10 },
      "701": { "keep" : 20 }
    },
    "evolve_all": "NONE",
    "evolve_speed": 20,
    "cp_min": 500,
    "use_lucky_egg": false,
    "evolve_captured": false,
    "catch": {
      "any": {"catch_above_cp": 0, "catch_above_iv": 0, "logic": "or"}
    },
    "release": {
        "any": {"release_under_cp": 500,"release_under_iv": 0.8, "cp_iv_logic": "and"},						
    "exceptions": {
        "always_capture": [
            "Arcanine",
            "Lapras",
            "Dragonite",
            "Snorlax",
            "Blastoise",
            "Moltres",
            "Articuno",
            "Zapdos",
            "Mew",
            "Mewtwo"
        ]
    }
    }
}
```
![image](https://cloud.githubusercontent.com/assets/8143724/17184783/5819ff1a-542e-11e6-9af4-a7c8e1b0bd64.png)

### Actual Behavior
nothing is released at start or during catching

### Steps to Reproduce
start latest dev

### Other Information
OS: Windows
Git Commit: (run 'git log -n 1 --pretty=format:"%H"' and paste it here)

**Comment by KLVN** [comment]
Check your `release_config.json`

**Comment by Muzurio** [comment]
i am on latest dev, no release_config.json anymore in there

**Comment by bimahp** [comment]
Can you try to change `"cp_iv_logic": "and"` to `"cp_iv_logic": "or"` ?

**Comment by Muzurio** [comment]
changed it to or, same result. It should be 'and' the rattat in the first print is under 500 AND under 0.8. So should be released

![image](https://cloud.githubusercontent.com/assets/8143724/17184957/2c358648-542f-11e6-9a3f-7dff6c918479.png)

**Comment by Ragekey** [comment]
#1151

**Comment by Ragekey** [comment]
My Bad, sorry wrong issue

**Comment by Muzurio** [comment]
@ragekey thats for EVOLVE, and i used the workaround for that and then changed back to original code. My issue is about RELEASE

**Comment by KLVN** [comment]
> i am on latest dev, no release_config.json anymore in there
whoops... Pepperidge Farm remembers :smile:

**Comment by matrix14** [comment]
change cp_min to 850

**Comment by Calcyfer** [comment]
In dev, `cp_min` was changed to `release_under_cp` , but I can confirm this is currently not working. Neither is transferring at start, etc. I guess it's all related.

**Comment by bimahp** [comment]
actually its now `"release_below_cp"` and `"release_below_iv"` as seen on example
just give it a shot

**Comment by Calcyfer** [comment]
@bimahp Thanks, damn these subtle changes! xD

**Comment by valerian** [comment]
The problem is that now the initial release is tied to the farming/catching release rules, I liked to be able to clean up all low CP pokemons every hour when I restarted the script.
I did a little edit to my code to have an `initial` category in the release rules, which is a global rule like `any` but is only taken in account in `initial_transfer_worker.py`, and as the highest priority.

Or maybe it's completely useless and there is something that I don't understand in the way I'm supposed to use the dev branch?
